src/core/drive/visit.ts

```diff
@@ -317,6 +317,8 @@ export class Visit implements FetchRequestDelegate {
       this.adapter.visitProposedToLocation(this.redirectedToLocation, {
         action: "replace",
         response: this.response,
+        shouldCacheSnapshot: false,
+        willRender: false,
```

Comment:
@seanpdoyle I understand why we don't want to render again, but I wonder if the flow should be the opposite?

For example, now what happens is the following
-> User clicks a link
-> Link redirects to somewhere else
-> Fetch API follows the redirect
-> Turbo then renders the response
-> Turbo then follows the redirect (updating the history, for example)
-> Turbo then doesn't render the response (because it doesn't want to render it twice)

But shouldn't it be something like
-> User clicks a link
-> Link redirects to somewhere else
-> Fetch API follows the redirect
-> Turbo then SKIPS rendering the response (because it is a redirect)
-> Turbo then follows the redirect (updating the history, for example)
-> Turbo then RENDERS the response

Following this flow, it would be more akin to what browsers do, which is first visiting the redirected location and then rendering the content of the page of that location, wdyt?

*NOTE*: My assumption may be wrong, and I am just misunderstanding how things work, but I thought that it is worth bringing it up anyway